Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue14763.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 14763, "[iOS] Resize on back-swipe stays when going back is cancelled", PlatformAffected.iOS)]
+	public class Issue14763 : NavigationPage
+	{
+		public Issue14763() : base(new HomePage())
+		{ }
+
+		class HomePage : ContentPage
```

Comment:
Could you include an UI Test here?.

Reply:
Yes, I will try to formulate an automated UI test as you suggested.